src/components/colors/Colors.module.scss

```diff
@@ -1,16 +1,16 @@
-.colors__color-block__item {
+.colorBlock {
   padding: 1rem;
   color: white;
```

Comment:
Should we enforce that all colors come from our design tokens, even for colors as basic as black and white?

Reply:
@JCarmine yes!